reference/3.0/Microsoft.PowerShell.Management/Set-Service.md

```diff
@@ -7,137 +7,135 @@ online version:  http://go.microsoft.com/fwlink/?LinkID=113399
+### Example 3: Change the description of a service
 
-The first command uses the Get-WmiObject cmdlet to get the Windows Management Instrumentation (WMI) object that represents the SysmonLog service.
-The default output of this command displays the start mode of the service.
+```powershell
+Get-CimInstance Win32_Service -Filter 'Name = "BITS"'  | Format-List  Name, Description
+```
 
-The second command uses Set-Service to change the start mode to automatic.
-Then, the first command is repeated to display the change.
+```output
+Name        : BITS
+Description : Transfers files in the background using idle network bandwidth. If the service is
+              disabled, then any applications that depend on BITS, such as Windows Update or MSN
+              Explorer, will be unable to automatically download programs and other information.
+```
 
-The final command displays the start mode of all services on the computer.
-### Example 3
+```powershell
+Set-Service -Name BITS -Description "Transfers files in the background using idle network bandwidth."
+Get-CimInstance Win32_Service -Filter 'Name = "BITS"'  | Format-List  Name, Description
 ```
-PS C:\> set-service -name Schedule -computername S1 -description "Configures and schedules tasks."
-PS C:\> get-wmiobject win32_service -computername s1 | where-object {$_.Name -eq "Schedule"} | format-list Name, Description
+
+```output
+Name        : BITS
+Description : Transfers files in the background using idle network bandwidth.
 ```
 
-These commands change the description of the Task Scheduler service on the S1 remote computer and then display the result.
+These commands change the description of the BITS service and then display the result.
 
-These commands use the Get-WmiObject cmdlet to get the Win32_Service object for the service, because the ServiceController object that Get-Service returns does not include the service description.
+These commands use the `Get-CimInstance` cmdlet to get the **Win32_Service** object for the service, because the **ServiceController** object that `Get-Service` returns does not include the service description.
 
-The first command  uses a Set-Service command to change the description.
-It identifies the service by using the service name of the service, "Schedule".
+### Example 4: Start a service
 
-The second command uses the Get-WmiObject cmdlet to get an instance of the WMI Win32_Service that represents the Task Scheduler service.
-The first element in the command gets all instances of the Win32_service class.
+```powershell
+Set-Service -Name "winrm" -Status Running -PassThru
+```
 
-The pipeline operator (|) passes the result to the Where-Object cmdlet, which selects instances with a value of "Schedule" in the Name property.
+This command starts the WinRM service.
+The command uses the Status parameter to specify the desired status, which is running, and the *PassThru* parameter to direct `Set-Service` to return an object that represents the WinRM service.
```

Comment:
https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/3-STYLE-GUIDE.md#formatting-syntax-elements

Property and parameter names should be in **bold** - eg **PassThru**